doc/en/warnings.rst

```diff
@@ -415,8 +415,20 @@ The following warning types ares used by pytest and are part of the public API:
 
 .. autoclass:: pytest.PytestWarning
 
-.. autoclass:: pytest.PytestDeprecationWarning
+.. autoclass:: pytest.PytestAssertRewriteWarning
 
-.. autoclass:: pytest.RemovedInPytest4Warning
+.. autoclass:: pytest.PytestCacheWarning
+
+.. autoclass:: pytest.PytestCollectionWarning
+
+.. autoclass:: pytest.PytestConfigWarning
+
+.. autoclass:: pytest.PytestDeprecationWarning
 
 .. autoclass:: pytest.PytestExperimentalApiWarning
+
+.. autoclass:: pytest.PytestUnhandledCoroutineWarning
+
+.. autoclass:: pytest.PytestUnknownMarkWarning
+
+.. autoclass:: pytest.RemovedInPytest4Warning
```

Comment:
RemovedInPytest4Warning is not used anymore really, is it?

Reply:
Ah, seen https://github.com/pytest-dev/pytest/issues/5180 now - but wonder why/if it is being kept now?

Reply:
I understand your point, but we really shouldn't remove that symbol in a minor release, it might break some configuration or other (it might be being referenced in a warnings filter for example).

It doesn't hurt keeping it for now and removing it on 5.0 to avoid breaking code needlessly.

Reply:
Makes sense, thanks for explaining!